util/dvsim/dvsim.py

```diff
@@ -409,16 +409,16 @@ def parse_args():
     waveg = parser.add_argument_group('Dumping waves')
 
     waveg.add_argument("--waves", "-w",
-                       action='store_true',
-                       help="Enable dumping of waves")
-
-    waveg.add_argument("-d",
-                       "--dump",
-                       choices=["fsdb", "shm", "vpd"],
-                       help=("Format to dump waves for simulation. The default "
-                             "format depends on the tool. With VCS, this "
-                             "defaults to fsdb if Verdi is installed, else "
-                             "vpd. With Xcelium, defaults to shm."))
+                       nargs="?",
+                       choices=["default", "fsdb", "shm", "vpd", "vcd", "evcd",
+                                "fst"],
+                       default=None,
+                       const="default",
+                       help=("Enable dumping of waves with the specified "
+                             "format. The default format depends on the tool. "
+                             "With VCS, this defaults to fsdb if Verdi is "
+                             "installed, else vpd. With Xcelium, defaults to "
```

Comment:
I think I'd get rid of this help text: the whole point of this PR seems to be to move stuff out of the Python, so we shouldn't keep describing the logic for specific simulators here, I don't think.

Reply:
Updated.